themes/roomify/roomify_travel/assets/scripts/roomify_travel.js

```diff
@@ -50,6 +49,21 @@ Drupal.behaviors.roomifyTravelScripts = {
         $(this).closest('.slide__caption').hide();
       }
     });
+
+    $('#roomify-main-menu-overlay .closebtn').once().click(function(e) {
+      //e.preventDefault();
```

Comment:
@salvoscala let's get rid of these commented lines? also a quick comment for what the code is doing would be nice :)